src/Core/src/Fonts/FontManager.Android.cs

```diff
@@ -171,7 +172,28 @@ public FontSize GetFontSize(Font font, float defaultFontSize = 0)
 			}
 
 			if (OperatingSystem.IsAndroidVersionAtLeast(28))
+			{
+				var fontWeightMap = new Dictionary<string, FontWeight>
+				{
+					{ "-light", FontWeight.Light },
+					{ "-medium", FontWeight.Medium },
+					{ "-black", FontWeight.Black }
+					// Add more styles as needed
+				};
+
+				if (!string.IsNullOrWhiteSpace(fontFamily))
+				{
+					foreach (var fontWeight in fontWeightMap.Keys)
+					{
+						if (fontFamily.EndsWith(fontWeight, StringComparison.OrdinalIgnoreCase))
+						{
+							return Typeface.Create(result, (int)fontWeightMap[fontWeight], italic);
```

Comment:
Then you wouldn't need to do a dictionary lookup here, you could call `.Value` on the key pair.